docs/java/java-project.md

```diff
@@ -9,49 +9,43 @@ MetaDescription: Lightweight Mode, Maven Support, Java Package and Dependency Ma
+The project view helps you view your Java projects and their dependencies, and provides entry points for project management tasks. 
 
 ![project view](images/java-project/projectmanager-overview.png)
 
-### Create a project
+In the overflow button of the navigation bar, there are more options to use. For example, you can switch between hierarchical view and flat view.
 
-You can create a project or source only workload by clicking the **+** sign on project view, or through the command: **Java: Create Java Project...**. During creation, VS Code will facilitate installing required extension(s) per your project type, if the extension(s) weren't installed.
+![overflow button](images/java-project/overflow-button.png)
 
-![create project](images/java-project/projectmanager-createproject.png)
+You can find buttons next to the nodes in the explorer, which can be used as useful shortcuts for some functionalities.
 
-### Import a project or module
+![inline button](images/java-project/inline-button.png)
 
-A project or module is imported to a workspace through **File** > **Open Folder** or **File** > **Open Workspace** menu. VS Code for Java will detect your project type automatically. As a tip, you can run the command **Java: Import Java projects in workspace** to reimport a project and alert the language server that there were changes to the project, without reloading your window.
+There are also many useful features in each node's context menu, you can right click on the node in explorer to toggle it out.
 
-### Add a dependency
+![context menu](images/java-project/context-menu.png)
 
-For Maven project, you can add a dependency by clicking the **+** sign next to **Maven Dependencies** node in project view.
+### Create Java project
 
-![add maven dependency](images/java-project/projectmanager-addmavendep.PNG)
+You can create a new Java project by clicking the **+** button in the navigation bar, or through the command: **Java: Create Java Project...** in Command Palette (`kb(workbench.action.showCommands)`). During creation, VS Code will facilitate installing required extension(s) per your project type, if the extension(s) weren't installed.
 
-### Add a JAR
+![create project](images/java-project/projectmanager-createproject.png)
 
-JAR file(s) can be added by clicking the **+** sign next to **Referenced Libraries** node in project view.
+### Import Java projects
 
-![Add Dependency](images/java-project/manage-dependencies.gif)
+Projects will be imported to workspace through **File** > **Open Folder...** (Make sure the opened folder contains your build tool scripts, i.e. `pom.xml` or `build.gradle`). VS Code for Java will detect your projects and import them automatically.
 
-For more details on library, refer to [Library Configuration](/docs/java/java-project.md#library-configuration).
+When you add a new module into your projects, you can trigger the command **Java: Import Java projects in workspace** to import them to workspace.
```

Comment:
we should probably say both project and modules can be imported using "Open Folder"